src/playerslist.js

```diff
@@ -149,4 +149,41 @@ export class PlayersList {
         // else
         this.players[0].isActive = true;
     }
+
+    getNextClockwise() {
+        let next = false;
```

Comment:
This needs to be deduplicated with the code above. Instead of having the same code copy/pasted in two places, have `setNextClockwise` call `getNextClockwise`